src/modules/previewpane/MonacoPreviewHandler/MonacoPreviewHandlerControl.cs

```diff
@@ -0,0 +1,260 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Drawing;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Windows.Forms;
+using Common;
+using Microsoft.PowerToys.PreviewHandler.Monaco.Properties;
+using Microsoft.Web.WebView2.Core;
+using Microsoft.Web.WebView2.WinForms;
+using Windows.System;
+using WK.Libraries.WTL;
+
+namespace Microsoft.PowerToys.PreviewHandler.Monaco
+{
+    public class MonacoPreviewHandlerControl : FormHandlerControl
+    {
+        /// <summary>
+        /// Settings class
+        /// </summary>
+        private readonly Settings _settings = new Settings();
+
+        /// <summary>
+        /// Saves if the user already navigated to the index page
+        /// </summary>
+        private bool _hasNavigated;
+
+        /// <summary>
+        /// WebView2 element
+        /// </summary>
+        private WebView2 _webView;
+
+        /// <summary>
+        /// WebView2 Environment
+        /// </summary>
+        private CoreWebView2Environment _webView2Environment;
+
+        /// <summary>
+        /// Loading label
+        /// </summary>
+        private Label _loading;
+
+        /// <summary>
+        /// Name of the virtual host
+        /// </summary>
+        public const string VirtualHostName = "PowerToysLocalMonaco";
+
+        [STAThread]
+        public override void DoPreview<T>(T dataSource)
+        {
+            base.DoPreview(dataSource);
+
+            // Starts loading screen
+            InitializeLoadingScreen();
+
+            // New webview2 element
+            _webView = new WebView2();
+
+            // Checks if dataSource is a string
+            if (!(dataSource is string filePath))
+            {
+                throw new ArgumentException($"{nameof(dataSource)} for {nameof(MonacoPreviewHandler)} must be a string but was a '{typeof(T)}'");
+            }
+
+            // Check if the file is too big.
+            long fileSize = new FileInfo(filePath).Length;
+
+            if (fileSize < _settings.MaxFileSize)
+            {
+                try
+                {
+                    InvokeOnControlThread(() =>
+                    {
+                        ConfiguredTaskAwaitable<CoreWebView2Environment>.ConfiguredTaskAwaiter
```

Comment:
`Awaitable` isn't currently being tagged here, but this is kind of why it's being flagged.

The other side is that `awaitable` is currently in `expect.txt`, but there are no longer any instances of it in lowercase, so the bot (with its current algorithm, which I think I've described earlier) is suggesting removing it (`awaitable`) and adding `Awaitable`.

The script would remove `awaitable` and add `Awaitable`.

Reply:
I added reported words to expect.txt and bot is still reporting the same 4 words...